test/karma/viewPrepareSpec.js

```diff
@@ -0,0 +1,106 @@
+var $ = require('jquery');
+var TorsoView = require('../../modules/View');
+
+describe('A View', function() {
+
+  describe('when using _prepare', function() {
+
+    it('can add context to the passed in argument', function() {
+      var PrepareView = TorsoView.extend({
+        initialize: function() {
+          this.set('bar', 444);
+        },
+
+        _prepare: function(context) {
+          context.foo = 123;
+        }
+      });
+      var prepareView = new PrepareView();
+      var context = prepareView.prepare();
+      expect(context.foo).toBe(123);
+      expect(context.view).toBeDefined();
+      expect(context.view.bar).toBe(444);
+    });
+
+    it('can override existing context', function() {
+      var PrepareView = TorsoView.extend({
```

Comment:
should this set bar in view state to make sure it is being overwritten?  A little bit ambiguous since you are overwriting something that doesn't have anything set on it.  Not a huge deal.

Reply:
I'm overridding the whole view object.